packages/common/src/reserved.ts

```diff
@@ -0,0 +1,31 @@
+export const TEMPORAL_RESERVED_PREFIX = '__temporal_';
+export const STACK_TRACE_RESERVED_PREFIX = '__stack_trace';
```

Comment:
That's not a prefix. Same for next line.

Reply:
renamed